sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MyApplication.java

```diff
@@ -16,13 +16,19 @@ public void onCreate() {
     // Example how to initialize the SDK manually which allows access to SentryOptions callbacks.
     // Make sure you disable the auto init via manifest meta-data: io.sentry.auto-init=false
     // SentryAndroid.init(
-    // this,
-    // options -> {
-    //   options.setBeforeSend(event -> {
-    //     event.setTag("sample-key", "before-send");
-    //   });
-    //   options.setAnrTimeoutIntervalMillis(2000);
-    // });
+    //    this,
+    //    options -> {
+    //      options.setBeforeSend(
+    //          (event, hint) -> {
+    //            event.setTag("sample-key", "before-send");
+    //            return event;
+    //          });
+    //      options.setAnrTimeoutIntervalMillis(2000);
+    //
+    // options.setDsn("https://d3964e5d568641c484c536600fb433a1@o552180.ingest.sentry.io/5677396");
```

Comment:
Whoops, it was meant to be the same DSN as in [AndroidManifest.xml](https://github.com/getsentry/sentry-java/blob/main/sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml#L37). Will change.

Reply:
we can actually disable the auto-mode for this sample and init. the SDK manually, so the samples for fragments would work as well, the other option would be creating a new sample that hooks up the fragment integration.

Reply:
I think it's a good call with disabling auto-mode here. I've enabled manual initialization in 513a3ec5a91f04bde6b5365a1f51af111d4220f0 - is this ok or you'd prefer to move setting `dsn` from `AndroidManifest` to `SentryAndroidOptions`?